lib/v2/ymgal/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/article': ['SunBK201'],
```

Comment:
```suggestion
    '/article/:type?': ['SunBK201'],
```
The key here is the same as the path in `router.js`